src/Honeybee_Honeybee.py

```diff
@@ -371,7 +371,6 @@ def downloadTemplates(self):
             except:
                 print 'Download failed!!! You need OpenStudioMasterTemplate.idf to use honeybee.' + \
                 '\nPlease check your internet connection, and try again!'
-                return -1
```

Comment:
@antonszilasi you're deleting a couple of lines that I recently added to Honeybee_honeybee. Can you get them fixed and send a new pull request?